### PR TITLE
https://github.com/stolostron/backlog/issues/27140

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/27140

### CODE REVIEW

File: clusters/hosted_control_planes/configure_hosted.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ Hosted control planes is a Technology Preview feature, so the related components
 [#hosting-service-cluster-configure]
 == Configuring the hosting cluster
 
-You can deploy hosted control planes by configuring an existing cluster to function as a hosting cluster. The hosting cluster is the {ocp} cluster where the control planes are hosted. Complete the following steps on the hub cluster where {mce} is installed to enable it as a hosting cluster:
+You can deploy hosted control planes by configuring an existing cluster to function as a hosting cluster. The hosting cluster is the {ocp} cluster where the control planes are hosted. The hosting cluster can be the hub cluster (also known as the `local-cluster`) or one of the {ocp-short} managed clusters. See the following topics to learn how to configure the `local-cluster` as the hosting cluster.
```
Comment (rokej) [inline clusters/hosted_control_planes/configure_hosted.adoc]: Remove _or one of the {ocp-short} managed clusters_.  ACM 2.7 supports only local-cluster.

File: clusters/hosted_control_planes/configure_hosted.adoc [hunk 2]
```diff
@@ -236,6 +232,41 @@ hypershift-addon   True
 
 Your HyperShift add-on is installed and the hosting cluster is available to create and manage HyperShift clusters.
 
+[#hosted-install-cli]
+=== Installing the hosted control planes CLI
+
+The hosted control planes CLI is an `oc` command line plugin that allows you to create and manage {ocp-short} hosted control plane clusters. After <<hosted-enable-feature,Enabling the hosted control planes feature>>, you can install the hosted control planes CLI by completing the following steps:
```
Comment (rokej) [inline clusters/hosted_control_planes/configure_hosted.adoc]: ~~The hosted control planes CLI is an `oc` command line plugin that allows you to create and manage {ocp-short} hosted control plane clusters.~~

The hosted control planes(HyperShift) CLI is used to create and manage {ocp-short} hosted control plane clusters.

**Comment by rokej** [comment]
@oafischer two comments. Otherwise, it looks good to me.

**Comment by oafischer** [comment]
Implemented the suggested changes, thanks!

**Pull request merged**